src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/Application.java

```diff
@@ -2,10 +2,8 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
 
 @SpringBootApplication
-@EnableCircuitBreaker
```

Comment:
Any reason why this was removed?

Reply:
Well, yes, it's because we don't use it, right? So just tried to reduce complexity a little and leave things that we do use. Happy to put it back if you consider we should, not a problem.